plugins/connection/docker.py

```diff
@@ -308,11 +330,34 @@ def put_file(self, in_path, out_path):
                 count = ' count=0'
             else:
                 count = ''
+            if getattr(self._shell, "_IS_WINDOWS", False):
+                cmd = '''& { $strm = [Console]::OpenStandardInput(1024)
+$data = [System.Array]::CreateInstance([byte],1024)
+$fstrm = [IO.File]::OpenWrite("%s")
+do {
+    $read = $strm.Read($data, 0, $data.Length)
+    if ($read -gt 0) {
+        $fstrm.Write($data, 0, $read)
+    }
+} while ($read -gt 0)
+$fstrm.Flush()
+$fstrm.Dispose()
+$strm.Dispose() }''' % (out_path)
+                args = self._build_exec_cmd(["cmd", "/c", self._shell._encode_script(cmd, as_list=False, strict_mode=False, preserve_rc=False)])
+                display.debug(u"Command to execute: {0}".format(args))
+            else:
+                out_path = self._prefix_login_path(out_path)
+                out_path = shlex_quote(out_path)
```

Comment:
I think these two lines should better be in another PR (with changelog fragment) - this looks like a bugfix (that should be backported), not like a new feature (like Windows container support, that should not be backported).

Reply:
I removed those two lines